docs/ABOUT.md

```diff
@@ -0,0 +1,11 @@
+# What is JavaScript?
+---
+Javascript is a scripting language used to provide dynamic and interactive content on webpages. It does not have class-oriented inheritance, but it does have prototype-oriented inheritance. Client-side JavaScript is commonly used. JavaScript provides benefits like Less server interaction, Immediate feedback to the site visitors, Increased interactivity and Richer interfaces.
+
+# Why should one learn JavaScript?
```

Comment:
Let's remove this header as well. The paragraph below it says it really well already.